### DISCUSSION TITLE
A export pt and load to DJL example

### DISCUSSION THREAD

**Discussion by weican** [deepjavalibrary/djl#1830] [Q&A]
I've trained retinaFace https://github.com/biubug6/Pytorch_Retinaface in colab. I also [converted](https://docs.djl.ai/docs/pytorch/how_to_convert_your_model_to_torchscript.html) the result of model pth file.
However, DJL shows the error: 
`torch::jit::load() received a file from torch.save(), but torch::jit::load() can only load files produced by torch.jit.save()
`
Does it have an example to guide me on how to train and export the correct pt model file?

**Comment by frankfliu** [comment]
DJL requires jit traced model. Please take a look: https://docs.djl.ai/master/docs/pytorch/how_to_convert_your_model_to_torchscript.html
